lisa/operating_system.py

```diff
@@ -119,6 +119,12 @@ class KernelInformation:
     version_parts: List[str]
 
 
+@dataclass
+class PackageInformation:
```

Comment:
Please modify existing `get_package_information`, make it compatible with any version format.

1. Create a class called `LisaVersionInfo`, which inherits from `VersionInfo`, so all existing cases should continue to pass.
2. Provide the `raw_version` field, so it can pass with non-semver.
3. Rewrite `parse_version` to return the new class.

Reply:
Done. Also refactored some related functions a bit. I'll need some help to validate that the change doesn't break any existing logic.